client/src/Models/Location.ts

```diff
@@ -0,0 +1,9 @@
+interface Location {
+    City: string,
+    Territory: string,
+    Country: string,
+    Longitude: number,
```

Comment:
Like the canada national stats model. Keep this one, ditch the backend one.

Reply:
I don't understand, doesn't having the backend one just make it really easy to send JSON that can be directly reconverted to a corresponding object on the front end? Why are we deleting back end models?